src/theme.tsx

```diff
@@ -2,21 +2,21 @@ import { createMuiTheme } from '@material-ui/core/styles';
 
 export default createMuiTheme({
   palette: {
-    primary: {
-      // light: will be calculated from palette.primary.main,
-      main: '#6C4087',
-      // dark: will be calculated from palette.primary.main,
-    },
-    secondary: {
-      // light: will be calculated from palette.primary.main,
-      main: '#C71C5B',
-      // dark: will be calculated from palette.primary.main,
-      // contrastText: will be calculated to contrast with palette.primary.main
-    },
-    text: {
-      //primary: '',
-      //secondary: ''
-    },
+    // primary: {
```

Comment:
What's this for?

Reply:
I think this was leftover from me playing around with the theming of the site, I can un-uncomment this

Reply:
Actually I wont, I'm going to leave it like this because we dont know what our color scheme should be yet and I think its best to leave it to MUI defaults. Currently the master branch has some theme styling that I put just to show how styling *could* be done. But honestly, looks kind of bad.